modules/recon/profiles-contacts/bing_linkedin_contacts.py

```diff
@@ -35,7 +36,9 @@ def get_contact_info(self, url):
             # Split the title on the pipe to get rid of "linkedIn" portion at the end
             name_and_title = link_title.split("|")[0]
             # Split whats left on the Dashes, which is usually name - title - company
-            name_title_company_list = name_and_title.split("-")
+            # some european LinkedIn sites use em-dash
+            DELIMETER_EXPRESSION = '- | ' + b'\xe2\x80\x93'.decode('utf-8')
```

Comment:
Apologies for a lack of clarity, I meant specifically moving the `b'\xe2\x80\x93'.decode('utf-8')` into a constant named something like `EM_DASH`. Though if you want to name the re expression, be my guest.